Tests/test_image_access.py

```diff
@@ -329,33 +329,36 @@ def test_p_putpixel_rgb_rgba(self):
 
 
 class TestImagePutPixelError(AccessTest):
-    def test_putpixel_error_message(self):
-        for mode, reason, accept_tuple in [
-            ("L", "color must be int or tuple", True),
-            ("LA", "color must be int or tuple", True),
-            ("RGB", "color must be int or tuple", True),
-            ("RGBA", "color must be int or tuple", True),
-            ("I", "color must be int", False),
-            ("I;16", "color must be int", False),
-            ("BGR;15", "color must be int", False),
-        ]:
-            im = hopper(mode)
-
-            for v in ["foo", 1.0, None]:
-                with pytest.raises(TypeError, match=reason):
-                    im.putpixel((0, 0), v)
-
-            if not accept_tuple:
-                with pytest.raises(TypeError, match=reason):
-                    im.putpixel((0, 0), (10,))
-
-            with pytest.raises(OverflowError):
-                im.putpixel((0, 0), 2 ** 80)
-
-        for mode in ["BGR;15"]:
-            im = hopper(mode)
-            with pytest.raises(ValueError, match="unrecognized image mode"):
-                im.putpixel((0, 0), 0)
+    IMAGE_MODES1 = ["L", "LA", "RGB", "RGBA"]
+    INVALID_TYPES1 = ["foo", 1.0, None]
```

Comment:
Group these together?
```suggestion
    IMAGE_MODES1 = ["L", "LA", "RGB", "RGBA"]
    IMAGE_MODES2 = ["I", "I;16", "BGR;15"]
    INVALID_TYPES1 = ["foo", 1.0, None]
    INVALID_TYPES2 = [*INVALID_TYPES1, (10,)]    
```